### DISCUSSION TITLE
Document could not be Loaded - Nothing on docs/api.json

### DISCUSSION THREAD

**Discussion by jcsix694** [dedoc/scramble#365] [Q&A]
Hi all, Looking for some help?

I had the docs working locally using sail/docker. I could also run sail in an EC2 container on AWS and get the docs.

I've since moved and made an actual production enviroment and all in docker. However I keep getting the message "Document could not be Loaded".

going to docs/api.json is reutning me nothing, where i expected the json like locally and on my EC2 container one.

Any pointers what I should be doing?

**Comment by romalytvynenko** [comment]
@jcsix694 

My first idea is to make sure that you've explicitly allowed Scramble to show documentation in non-local environment (https://scramble.dedoc.co/usage/security)?

Also, is the error reproducible locally when you change `APP_ENV` to `production`?
